Document/0x05b-Basic-Security_Testing.md

```diff
@@ -59,12 +59,86 @@ The first step in rooting a mobile is to unlock its boot loader. The procedure d
 
 The available setup options for the network need to be evaluated first. The mobile device used for testing and the machine running the interception proxy need to be placed within the same WiFi network. Either an (existing) access point is used or [an ad-hoc wireless network is created](https://support.portswigger.net/customer/portal/articles/1841150-Mobile%20Set-up_Ad-hoc%20network_OSX.html "Creating an Ad-hoc Wireless Network in OS X").
 
-Once the network is configured and connectivity is established between the testing machine and the mobile device, several other steps need to be done.
+Once the network is configured and connectivity is established between the testing machine and the mobile device, next steps are configuring a proxy and installing a CA certificate.
 
-- The proxy in the network settings of the Android device need to be [configured properly to point to the interception proxy in use](https://support.portswigger.net/customer/portal/articles/1841101-Mobile%20Set-up_Android%20Device.html "Configuring an Android Device to Work With Burp").
-- The [CA certificate of the interception proxy need to be added to the trusted certificates in the certificate storage of the Android device](https://support.portswigger.net/customer/portal/articles/1841102-installing-burp-s-ca-certificate-in-an-android-device "Installing Burp's CA Certificate in an Android Device"). Due to different versions of Android and modifications of Android OEMs to the settings menu, the location of the menu to store the CA certificate might differ.
+###### Configuring Proxy
 
-After finishing these steps and starting the app, the requests should show up in the interception proxy.
+A proxy allows for intercepting and modifying network requests and responses. The configuration procedure will be explained with a usage of [Burp Suite free edition](https://portswigger.net/burp/freedownload).
```

Comment:
> A proxy allows for intercepting

I think using "allows for" is not entirely correct. "Allows for" means having some leeeway in a plan. Maybe you can make it something like this:

> A proxy can intercept and modify network requests and responses.

Reply:
Thanks!